src/rust/ide/src/model/synchronized/graph/executed.rs

```diff
@@ -11,6 +11,25 @@ use crate::model::execution_context::VisualizationId;
 use crate::model::execution_context::VisualizationUpdateData;
 use crate::model::synchronized::ExecutionContext;
 
+use enso_protocol::language_server::MethodPointer;
+use flo_stream::MessagePublisher;
+
+
+
+// ==============
+// === Errors ===
+// ==============
+
+#[allow(missing_docs)]
+#[fail(display = "The node {} has not been evaluated yet.", _0)]
```

Comment:
We can use simpler form here: "The node {} di not evaluate yet." . Confirm with @iamrecursion please.

Reply:
What is written in the code is correct.